tests/suites/os/tests/secureboot/index.js

```diff
@@ -1,145 +1,285 @@
 'use strict';
 
-module.exports = {
-	title: 'secure boot tests',
-	tests: [
-		{
-			title: 'check secure boot',
-			run: async function(test) {
-				const securebootEfiVarPath = '/sys/firmware/efi/efivars/SecureBoot-*';
+import { mkdtemp } from 'node:fs/promises';
```

Comment:
These imports will break leviathan - we can remove them , as we're not going to use these modules anyway (see above comment about wanting to create temp directory on the DUT

Reply:
Removed them